src/status_im/contexts/wallet/sheets/buy_token/view.cljs

```diff
@@ -21,14 +23,38 @@
     :image-props       {:icon logo-url}
     :on-press          #(rn/open-url site-url)}])
 
+(def ^:private tabs
+  [{:id    :one-time
+    :label "One time"}
+   {:id    :recurrent
+    :label "Recurrent"}])
+
+(def ^:private initial-tab (:id (first tabs)))
+
 (defn view
   []
   (rn/use-mount (fn []
                   (rf/dispatch [:wallet/get-crypto-on-ramps])))
-  (let [crypto-on-ramps (rf/sub [:wallet/crypto-on-ramps])]
+  (let [crypto-on-ramps                 (rf/sub [:wallet/crypto-on-ramps])
+        crypto-recurrent                []
+        [selected-tab set-selected-tab] (rn/use-state initial-tab)
+        [min-height set-min-height]     (rn/use-state 0)
+        on-layout                       (rn/use-callback
+                                         #(set-min-height
+                                           (oops/oget % :nativeEvent :layout :height)))]
```

Comment:
Just wondering why we need to use the on-layout and min-height. Is this intended to maintain the same height for the `recurrent` tab (empty for now) as the `one-time` tab?

Reply:
Yes. All the platforms may not support recurrent purchases and we want to preserve the drawer height when changing tabs.